ui/app/mainui/AppMain.qml

```diff
@@ -1276,4 +1196,83 @@ Item {
             keycardPopup.item.open()
         }
     }
+
+    DropArea {
```

Comment:
@alexandraB99, please exclude it to a separate file. We need to reduce size of this file step by step :)